README.md

```diff
@@ -60,7 +60,10 @@ For requestor the app should implement a long running RPC service which implemen
   - Shouldn't take much time (preferably less than a minute for medium range machines).
 - `AbortTask`
   - Takes one argument: `task_id`.
-  - Will be called when the task is aborted by the user. Should stop all running subtask verifications for this task and perform any other necessary cleanup.
+  - Will be called when the task is aborted by the user or timed out. Should stop all running subtask verifications for this task and perform any other necessary cleanup.
+- `AbortSubtask`
+  - Takes two arguments: `task_id` and `subtask_ids`.
+  - Will be called when the subtask is aborted by the user timed out. Should stop verification of the subtask (if it's running) and perform any other necessary cleanup.
```

Comment:
Should it stop the verification? then there is a chance the subtask will get assigned again

Reply:
I thought that's what we agreed on.

Reply:
yes, we agreed on not calling the app_client from RTM when the status is verifying.

Just when reading it i thought about `get_next_subtask` that will get a race condition